client/easy_to_forget_to_dos.txt

```diff
@@ -0,0 +1,2 @@
+- initial input field changes on focus: blue haze, or transform/grow like material gets closer.
+- put 'chosen address' in redux
```

Comment:
Are these todos supposed to be done in this PR?

Reply:
Nah, this file is super old, from before we had waffle I think. I just deleted it.